### PR TITLE
[ET-1520] Fix - Avoid loading partner JS everywhere on WP Admin

### PR DESCRIPTION
**Pull request by rafsuntaskin**
### 🎫 Ticket

[ET-1520] 

### 🗒️ Description

* PayPal's signup partner JS asset was getting loaded on every page of the admin.

 📷 screenshot: https://d.pr/i/vrKRq2

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1520]: https://theeventscalendar.atlassian.net/browse/ET-1520?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by juanfra — COMMENTED** [review]
Looks good, let's add a readme entry for this (now that we have a ticket).

**Comment by rafsuntaskin** [comment]
> Looks good, let's add a readme entry for this (now that we have a ticket).

I am waiting for the `release/G22.leptoceratops` to go out before I update the readme for all the PRs for next MR.

**Comment by juanfra** [comment]
@rafsuntaskin perfect! Leptoceratops was released so we probably can update this one now.

File: src/Tickets/Commerce/Gateways/PayPal/Assets.php [hunk 1]
```diff
@@ -36,6 +36,7 @@ public function register() {
 			[],
 			'admin_enqueue_scripts',
 			[
+				'conditionals' => [ $this, 'should_enqueue_assets_payments_tab' ],
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/PayPal/Assets.php:39]: Should we also modify the `should_enqueue_assets_payments_tab` to check if we're on the `paypal` tc-section? (now that we have one).

Probably this logic was implemented before we had "sections" on the Settings. And if we don't include that check, the assets will be loaded on other settings and in the payments tab in general.

Comment (rafsuntaskin) [reply src/Tickets/Commerce/Gateways/PayPal/Assets.php:39]: Correct, we should do that. Updated with the logic now.

**Review by juanfra — APPROVED** [review]

**Pull request merged**